salesagility/SuiteCRM
# Not translatable values in Activities/Popup_picker.php - Additional

Additional Fix for Issue https://github.com/salesagility/SuiteCRM/issues/3504
Following from PR https://github.com/salesagility/SuiteCRM/pull/3505



## Description




## Motivation and Context


## How To Test This


1.  Install language pack
2. Apply additional translations to new strings (see develop branch)
3. See the translated strings are visible on the View Summary popup.

## Types of changes


### Final checklist
